### ISSUE TITLE
Setting to automatically fetch metadata when following a new person 

### ISSUE THREAD

**Issue by Nate-Levin** [mikedilger/gossip#131]
Would be nice to have a setting that determined whether or not to automatically fetch a persons metadata when newly followed.

**Comment by Nate-Levin** [comment]
I will look at implementing this tonight. This should probably be false by default

**Comment by mikedilger** [comment]
reasonable.

**Comment by mikedilger** [comment]
This could be a trigger of overlord::refresh_followed_metadata, or just for the one person followed, so long as it doesn't happen over and over in a loop (e.g. in people::async_follow, if you did it, that might be called by other functions many times and trigger pulling metadata over and over).  Anyhow, it can be worked out.

**Comment by mikedilger** [comment]
The problem with this is that when you follow a new person, it has to subscribe for their relay list events, then only after those come in and get processed will it know where to get metadata.  We don't have a nice async coding system to remember what we were doing, so this is hard at present.  I think it's okay as it is for now.

Bumping to milestone 0.5

**Comment by mikedilger** [comment]
Actually now that we automatically fetch metadata when we encounter in the UI someone without it, this is now resolved.
